frontend/src/citizen-frontend/applications/editor/validations.ts

```diff
@@ -92,12 +92,16 @@ export const isValidDecisionStartDate = (
   startDate: string,
   type: DecisionType
 ): boolean => {
-  return date !== null
-    ? date.isEqualOrAfter(LocalDate.parseFi(startDate)) &&
-        date.isEqualOrBefore(
-          maxDecisionStartDate(LocalDate.parseFi(startDate), type)
-        )
-    : false
+  try {
+    return date !== null
```

Comment:
signature says `date: LocalDate` so it cannot be null

Reply:
True, removed the extra null check from `isValidPreferredStartDate` as well.